### PR TITLE
Adding script for multiband paper experiments

### PR DESCRIPTION
**Pull request by sawanp813**
This PR adds a script, `multiband_exps.sh`, which runs experiments which are used to generate the ablation and classification detection figures from _Bliss for multiband images_. 

NOTE: Have not yet tested if script runs end-to-end (i.e. first training three models and then running experiments), but it should work with minor modifications to `multiband_exp.ipynb` notebook. 

Current issues:
- Plots for galaxy classification figure look off w.r.t. axes ticks and cosmetic things.

### CODE REVIEW

**Review by jeff-regier — CHANGES_REQUESTED** [review]
It looks like `multiband_exps.sh` may have been omitted?

Please lmk when that script runs end-to-end.

File: case_studies/multiband/multiband_exps.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+# train five-band model
+bliss -cp ~/bliss/case_studies/multiband -cn config mode=train \
+    train.trainer.logger.name=MULTIBAND \
+    train.trainer.logger.version=ugriz_band \
+    encoder.image_normalizer.bands=[0,1,2,3,4] \
+    cached_simulator.cached_data_path=/data/scratch/regier/2percent
+
+# run the evaluation notebook
+jupyter nbconvert --execute multiband_exp.ipynb --to html
```
Comment (jeff-regier) [inline case_studies/multiband/multiband_exps.sh:27]: Should we execute the `alignment.ipynb` and `color_prior.ipynb` notebooks here too? (Are results from these notebooks included in the manuscript?)

File: case_studies/multiband/multiband_exps.sh [hunk 2]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Runs the experiments for the BLISS multiband paper.
+
+# train single-band model
+bliss -cp ~/bliss/case_studies/multiband -cn config mode=train \
+    train.trainer.logger.name=MULTIBAND \
+    train.trainer.logger.version=r_band \
+    encoder.image_normalizer.bands=[2] \
+    cached_simulator.cached_data_path=/data/scratch/regier/2percent
```
Comment (jeff-regier) [inline case_studies/multiband/multiband_exps.sh:10]: Would you make `cached_data_path` a command line argument to make that is set to `/data/scratch/regier/2percent` by default, but which can be modified?

File: case_studies/multiband/multiband_exps.sh (hunk 1 above)
Comment (sawanp813) [reply case_studies/multiband/multiband_exps.sh:27]: Sure I'll have those added to the script as well.

**Comment by jeff-regier** [comment]
@sawanp813 Is this ready to merge?

**Comment by sawanp813** [comment]
> @sawanp813 Is this ready to merge?

I haven't run the script recently following the recent changes. I can verify everything checks out in the coming days, though if I remember correctly there were some issues with metrics that seemed off.

**Pull request merged**